cmd/src/users.go

```diff
@@ -16,11 +16,12 @@ Usage:
 
 The commands are:
 
-	list       lists users
-	get        gets a user
-	create     creates a user account
-	delete     deletes a user account
-	tag        add/remove a tag on a user
+	list          lists users
+	get           gets a user
+	create        creates a user account
+	delete        deletes a user account
+	tag           add/remove a tag on a user
+	verify-emails verify user emails in bulk
```

Comment:
If we do go with this approach, this should be done as an `emails` subcommand not as an `verify-emails` command. This enables us in the future to offer other things like:

- `src users emails add -username=alice alice@gmail.com`
- `src users emails remove -username=alice alice@gmail.com`

etc.